models/snapchat.yml

```diff
@@ -9,12 +9,45 @@ models:
             - ad_account_id
             - date_day
     columns:
+      - name: platform
+        description: Advertising platform.
       - name: date_day
         description: The date of the report.
       - name: ad_account_id
         description: The ID of the account in Snapchat.
       - name: ad_account_name
         description: The name of the account in Snapchat.
+      - name: currency
+        description: The current used by the account in Snapchat.
+      - name: spend
+        description: The spend on the ad in the given day.
+      - name: impressions
+        description: The number of impressions the ad had on the given day.
+      - name: swipes
+        description: The number of swipes the ad had on the given day.
+
+  - name: snapchat__ad_report
+    description: Each record represents the daily performance of a Snapchat ad account.
```

Comment:
This is the same description as the `snapchat__account_report` model. Can we add the proper description for this model.